.github/workflows/webapp_check.yml

```diff
@@ -0,0 +1,21 @@
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-10.15, ubuntu-18.04]
+        steps:
+          - name: Check out Github repository
+            uses: actions/checkout@v2
+          - name: Set up Node
+            uses: actions/setup-node@v3
+            with:
+              node-version: '16'
+          - name: Compile
+            run: npm run build --prefix webapp
+          - name: Check style
+            run: npm run style --prefix transcendence-app
```

Comment:
I think this should be `npm run style --prefix webapp`

Reply:
same as above, thanks!